website_chd_product_configurator/static/src/js/website_chd_product_configurator.js

```diff
@@ -0,0 +1,121 @@
+$(document).ready(function () {
+
+	$('.type_selection').on('change', function ()
+    {
+    	openerp.jsonRpc('/chd_init/getch/', 'call',
+    		 {'type_id': this.value,}).then(function (data)
+    				 {
+    			            $('#fini_select_id').empty();
+    			  			$('#fini_select_id').append("<option></option>");
+    			  			var data_js = eval(data);
+    			  			$.each(data_js,function(element)
+    			  			{
+    			  				if(data_js[element])
+    			  				{
+    			  					$('#fini_select_id').append("<option value='" + data_js[element].id + "'>"+ data_js[element].name + "</option>");
+    			  				}
+
+    			  			});
+    		  })
+    });
+
+	//	alternative proposed that does not use json unfortunately i get
+	//msg openerp.web undefined.
+	/*$('.type_selection').on('click', function ()
```

Comment:
this proposal came from being used to program for the backend, there, it's slightly different. Apologies for the confusion